fvm/state/transaction_state.go

```diff
@@ -198,28 +223,51 @@ func (s *TransactionState) CommitParseRestricted(
 		)
 	}
 
-	err := s.mergeIntoParent()
-	if err != nil {
-		return nil, err
+	return s.mergeIntoParent()
+}
+
+// Pause detaches the current nested transaction from the parent transaction,
+// and returns the paused nested transaction state.  The paused nested
+// transaction may be resume via Resume.
+//
+// WARNING: Pause and Resume are intended for implementing continuation passing
+// style behavior for the transaction invoker, with the assumption that no
+// other transaction processor modify the state (other than the spock and
+// metering). The paused nested transaction should not be reused across
```

Comment:
currently the proposer sequence number is modified during the verify tx processor.

Reply:
thanks for pointing this out.  i think we're ok in this case since program parsing shouldn't read the sequence number.   I'll update the comment and keep a look out for this.